README

```diff
@@ -40,17 +42,36 @@ GNU versions of these tools are recommended.
 
 Installation
 ------------
-Installing Geany is done by the following three commands:
-$ ./configure
-$ make
-(as root)
-% make install
 
-If you are building from a Git clone rather than a source tarball, you need
-to run ./autogen.sh first.
+To build the user manual you need *rst2html* from Docutils. A pre-built
+version is available in distribution tarballs and will be used as
+fallback if *rst2html* is missing. When building from Git however, that
+pre-built version is not included and *rst2html* is required by default.
+You can explicitly disable building the user manual using the
+``--disable-html-docs`` *configure* flag, but this will result in not
```

Comment:
because it's a bad thing nobody should want.  at least that's what we currently have decided.

And because we want it for creating tarballs, although we could very well force it just in this case.